packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -206,7 +207,9 @@ class Uploader {
   }
 
   async _uploadByProtocol () {
-    const { protocol } = this.options
+    // todo a default protocol should not be set. We should ensure that the user specifies their protocol.
+    // after we drop old versions of uppy client we can remove this
```

Comment:
todo no longer relevant.

Reply:
i think it's still relevant, because we might want to do this in the future if we ever break backward compatibility with old uppy clients. I want to explicitly add `protocol: 'multipart'` in the client also

Reply:
Alright!